src/main/java/com/google/sps/servlets/schedule/UserPreferences.java

```diff
@@ -38,30 +43,54 @@
     public static int EVENT_RECURRENCE_LENGTH = 4; // in weeks
 
     // Possible study event sessions
-    public Map<Integer, Integer> STUDY_SESSION_START_TIME = new LinkedHashMap<Integer, Integer>();
-    public Map<Integer, Integer> STUDY_SESSION_LENGTH = new LinkedHashMap<Integer, Integer>();
+    public static Map<Integer, Integer> STUDY_SESSION_START_TIME = new LinkedHashMap<Integer, Integer>() {{
```

Comment:
Why are we using a map here? Usually maps are used for categorizing or labeling things, not so much for separating values.

Reply:
Great point! 

The first map is the possible start times that a study session can start at and the other map is the duration. The first value represents the hour and the second value represents the minute. 

What would be better is to use a tuple to hold the hour and minute pair, but I'm not sure if Java has tuples without importing a library so we kind of get around that by using a hash map to hold our pairs. It's definitely not as readable as it could be and could cause confusion.